openedx/core/djangoapps/appsembler/sites/management/commands/reindex_elasticsearch.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+
+from contentstore.views.course import reindex_course_and_check_access
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+class Command(BaseCommand):
+    help = "Force re-index of elasticsearch for all courses"
+
+    def handle(self, *args, **options):
+        all_courses = CourseOverview.get_all_courses()
```

Comment:
@thraxil I think what you're looking for is something like this:

https://github.com/appsembler/edx-platform/blob/a09a81b138d722b5a4a211f21ef85e0ec9b02098/common/djangoapps/enrollment/tests/test_views.py#L155


Maybe
```suggestion
        if not settings.ROOT_URLCONF == 'cms.urls':
           raise CommandError('lorem ipsum')
        all_courses = CourseOverview.get_all_courses()
```

Reply:
Done. Thanks.